tests/ttnn/integration_tests/stable_diffusion/test_unet_2d_condition_model_new_conv.py

```diff
@@ -144,8 +148,6 @@ def test_unet_2d_condition_model_256x256(device, batch_size, in_channels, input_
     ],
 )
 def test_unet_2d_condition_model_512x512(device, batch_size, in_channels, input_height, input_width):
-    device.enable_program_cache()
```

Comment:
We still want program cache.

Reply:
Added back.

Reply:
Enabling program cache is causing a crash.

Reply:
The hang needs to be debugged, we cannot enable new conv API without program cache.